tests/test_app.py

```diff
@@ -170,7 +170,7 @@ def test_app_day_after_gsp_only_after_national(db_connection, input_data_last_up
     with db_connection.get_session() as session:
         gsps = session.query(LocationSQL).all()
         _ = Location.from_orm(gsps[0])
-        assert len(gsps) == 6
+        assert len(gsps) == 5
 
         gsp_yields = session.query(GSPYieldSQL).all()
-        assert len(gsp_yields) == 6 * 49  # 5 gsps with 48 half hour settlement periods + midnight
+        assert len(gsp_yields) == 5 * 49  # 4 gsps with 48 half hour settlement periods + midnight
```

Comment:
And this number doesn't make sense with the comment either

Reply:
Thanks, ive updated it, it include national, so its 4 gsp + national to make 5